### ISSUE TITLE
c-lightning support

### ISSUE THREAD

**Issue by LightningHelper** [apotdevin/thunderhub#336]
c-lightning support

**Comment by TheCherry** [comment]
that will be nice!

**Comment by cryptoquick** [comment]
Yeah, the new Rust RPC is really cool to see. One thing I've been noticing, Lightning Labs is really toxic to the broader Lightning community, so maybe it's hopeless to ask for CLN support. Similarly, who knows if they'll support BOLT-12 in LND, considering it competes with the Pool business model.
